### PR TITLE
doc: Add minutes for NJSI 2018 Collab Summit

### PR DESCRIPTION
**Pull request by digitalinfinity**

### CODE REVIEW

**Comment by digitalinfinity** [comment]
cc @addaleax @codebytere @gabrielschulhof @jschlight @mafintosh @mhdawson @NickNaso @ofrobots

Please let me know if I got something wrong/missed something

There were also other folks there in the room whose names I don't know- if the attendee list needs to be updated with additional GitHub handles, please let me know

Thanks everyone for a very productive conversation at Collab Summit! Great catching up with you all!

**Review by NickNaso — APPROVED** [review]
LGTM

File: meeting notes/2018-10-13-NJSI-CollabSummit.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+## WASM/Native modules (led by @ofrobots)
+@ofrobots:
+- Thinking of kick-starting a team or effort to make WASM native modules a reality.
+- Few use cases where N-API is not needed
+- Not all use cases for N-API covered by WASM but there are some
+- Canonical example: node-sass- every time new node version comes out, need to ship new binaries. It's pure computation that can have it ship as a native module
+- Would be good to have a team to work through the challenges
+    - Don’t have a good toolchain - emscripten is from a web perspective. If you want to use malloc, where would it come from? Emscripten- ship something like musl-c - this is not great, especially if you have multiple WASM modules
+    - Would be cool if node provided an LLCM wrapper out of the box, variant of Emscripten for Node
```
Comment (ofrobots) [inline meeting notes/2018-10-13-NJSI-CollabSummit.md]:
```suggestion
    - Would be cool if node provided an LLVM wrapper out of the box, variant of Emscripten for Node
```

**Review by ofrobots — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

**Comment by codebytere** [comment]
approved fwiw :)

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by NickNaso** [comment]
Could be possible merge this?

**Comment by digitalinfinity** [comment]
Landed in a4ece21dd776c9e0dc62a1adbcfdf606c050fb1d. Thanks everyone!

**Pull request closed without merging**